api/nodes/serializers.py

```diff
@@ -102,10 +104,11 @@ class NodeSerializer(JSONAPISerializer):
         related_meta={'count': 'get_pointers_count'},
     )))
 
-    parent = HideIfRetraction(RelationshipField(
```

Comment:
I know these are due to an underlying bug in the api. I'm okay with this as long as @sloria is, but please make a note of the side effects in the PR description.